manifest.php

```diff
@@ -29,7 +29,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '10.11.1',
+    'version'     => '10.11.2',
```

Comment:
You should increase the feature number since you've added a new feature. 
You should also increase the dependency to the tao extension.